examples/Example16-AccessMultiple/Example16-AccessMultiple.ino

```diff
@@ -0,0 +1,163 @@
+/*
```

Comment:
Hi @guihomework ,
Thank you for submitting this PR.
Some changes are required...
We already have an Example16. Please change the title of this file to Example18-AccessMultiple.ino.
We have separate examples for I2C and SPI. Please change this example so it uses I2C. You can include a separate SPI example in the SPI\examples folder.
Please comment out the Teensy-specific setMOSI commands etc.. Please see the other SPI examples and use the same format - using generic SPI. It is OK to leave the Teensy code in - so long as it is commented.
I think you missed an '&'? (See comment below.
Thank you!
Paul

Reply:
This is a nice PR. Great work!
Please make the suggested changes and I will be happy to merge it.
Best wishes,
Paul